### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by CodeMan99**
Whoops! I missed this when I did PR #43 

Also, what your policy on updating dependencies in general? There are a few updates that could be applied.

### CODE REVIEW

**Comment by kevva** [comment]
You could update the dependencies in this PR too as long as they don't require a new semver-major.

**Comment by CodeMan99** [comment]
Still outstanding updates:

```
$ npm outdated
Package     Current  Wanted  Latest  Location
p-event       1.3.0   1.3.0   2.1.0  gulp-debug
strip-ansi    4.0.0   4.0.0   5.0.0  gulp-debug
through2      2.0.5   2.0.5   3.0.0  gulp-debug
```

I am pretty sure these are all just minimum support node versions, but wanted to double check before moving one.

File: package.json [hunk 1]
```diff
@@ -30,19 +30,19 @@
 		"fs"
 	],
 	"dependencies": {
-		"chalk": "^2.3.0",
```
[the remark this answers is not in the document]
Comment (CodeMan99) [reply package.json]: Minimum node version for gulp-debug is v6.x?

Also, are you saying to not make changes to the caret ranges? I'm not following the reasoning behind that.

**Comment by kevva** [comment]
Thanks @CodeMan99 :)

**Pull request merged**